<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditModeDockWidget</class>
 <widget class="QDockWidget" name="EditModeDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Mode</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Mode</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="RadioButton_LayerMode">
         <property name="text">
          <string>Layer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RadioButton_EntityMode">
         <property name="text">
          <string>Entity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RadioButton_DoorMode">
         <property name="text">
          <string>Door</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RadioButton_CameraMode">
         <property name="text">
          <string>Camera</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>View</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="CheckBox_Layer0View">
         <property name="toolTip">
          <string>Layer 0</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_Layer1View">
         <property name="toolTip">
          <string>Layer 1</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_Layer2View">
         <property name="toolTip">
          <string>Layer 2</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_Layer3View">
         <property name="toolTip">
          <string>Layer 3</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_EntityView">
         <property name="toolTip">
          <string>Entities</string>
         </property>
         <property name="text">
          <string>E</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_DoorView">
         <property name="toolTip">
          <string>Doors</string>
         </property>
         <property name="text">
          <string>D</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_AlphaView">
         <property name="toolTip">
          <string>Alpha</string>
         </property>
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckBox_CameraView">
         <property name="toolTip">
          <string>Camera</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Layer now edit on</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="RadioButton_EditOnLayer0">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RadioButton_EditOnLayer1">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RadioButton_EditOnLayer2">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RadioButton_EditOnLayer3">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
